build_all/base-libraries/AzureIoTHub/README.md

```diff
@@ -101,22 +100,23 @@ You should have the following ready before beginning with any board:
 
 3. Update Wifi SSID/Password in `iot_configs.h`
 
-    - Ensure you are using a wifi network that does not require additional manual steps after connection, such as opening a web browser.
+- Ensure you are using a wifi network that does not require additional manual steps after connection, such as opening a web browser.
 
 4. Update IoT Hub Connection string in `iot_configs.h`
 
 5. Navigate to where your esp32 board package is located, typically in `C:\Users\<your username>\AppData\Local\Arduino15\packages` on Windows and `~/.arduino15/packages/` on Linux
 
-- Locate the board's `platform.txt` and add the define `-DDONT_USE_UPLOADTOBLOB` on line 53 (`build.extra_flags=`) 
+	- Navigate deeper in to `hardware/esp8266/<board package version>/` where the `platform.txt` file lives.
 	
-6. Navigate to the AzureIoTUtility library location, typically `C:\Users\<your username>\Documents\Arduino\libraries\AzureIoTUtility\src` on Windows, and `~/Arduino/libraries/AzureIoTUtility/src` on Linux
-- Once there, ensure that `tlsio_mbedtls.c` is in the `adapters` folder.
-
-7. Run the sample.
+	- Copy the `platform.local.esp32.txt` file from the sample to the same folder as the `platform.txt` file, and rename it `platform.local.txt`.
```

Comment:
could this instead be managed in a way such that you follow something like this:

> Select the `platform.local.txt` from the `esp32/` subdirectory and copy it to the same directory as the `platform.txt`.

Might be easier to manage from a user perspective.

Reply:
Yeah, I agree, this makes it so you don't have to rename them, done.